yt/yt/client/api/operation_client.h

```diff
@@ -26,6 +26,7 @@ struct TSuspendOperationOptions
     : public TTimeoutOptions
 {
     bool AbortRunningJobs = false;
+    TString Reason;
```

Comment:
I prefer to make this parameter optional

Reply:
Do you want it to still be `optional string reason = 4;` in protobuf and empty in this case?

Reply:
I do not clearly understand what do you mean.

I think that `std::nullopt` semantically much better as a default value than `""`

Reply:
I am asking, how should we distinguish missing reason from an empty one, when passing them along the network in protobuf.

Reply:
protobuf has `has_*` method to check value presence

Example: https://github.com/ytsaurus/ytsaurus/blob/main/yt/yt/server/lib/rpc_proxy/api_service.cpp#L178

Reply:
Is not it impossible to distinguish a missing string from an empty one?
https://protobuf.dev/programming-guides/field_presence/

Reply:
We use proto2 syntax and in this case missing string and specified empty string are distiguishable (as far as i know)